Moved Deployment creation steps from "Deploy via CLI" to "Configure a Deployment" (#211)
astronomer/docs: cloud/stable/03_deploy/03_deploy-cli.md

Before you deploy to Astronomer, make sure that the Airflow Deployment you'd like to deploy to is within the Workspace you're operating in.

To see the list of Workspaces you have access to, run:

```sh
$ astro workspace list
```

To switch between Workspaces, run:

```sh
$ astro workspace switch
```

To see the list of Deployments within a particular Workspace, run:

```sh
$ astro deployment list
```

For more specific CLI guidelines and commands, read [CLI Quickstart](/docs/cloud/stable/develop/cli-quickstart/).

## Step 3: Deploy to Astronomer

Finally, make sure you're in the correct Airflow project directory.

When you're ready to deploy your DAGs, run:

```sh
$ astro deploy
```

This command returns a list of Airflow Deployments available in your Workspace and prompts you to pick one.

## Step 4: Validate Your Changes

If it's your first time deploying, expect to wait a few minutes for the Docker Image to build.

To confirm that your deploy was successful, navigate to your Deployment in the Astronomer UI and click **Open Airflow** to see your changes in the Airflow UI.

### What gets deployed?

> Everything in the project directory where you ran `$ astro dev init` is bundled into a Docker image and deployed to your Airflow Deployment on Astronomer Cloud. We don't deploy any of the metadata associated with your local Airflow environment (e.g. task history, Airflow Connections and Variables set in the Airflow UI, etc.) - just the code.

Reviewer: ```suggestion
Everything in the project directory where you ran `$ astro dev init` is bundled into a Docker image and deployed to your Airflow Deployment on Astronomer Cloud. Astronomer exclusively deploys the code in your project and does not push any of the metadata associated with your local Airflow environment, including task history and any Airflow Connections or Variables set locally in the Airflow UI.
```
